src/Components/GithubCard.css

```diff
@@ -0,0 +1,51 @@
+.box-header {
+    display: flex;
+    align-items: center;
+    height: 68px;
+    justify-content: space-between;
+    padding-bottom: 16px;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-top: 16px;
+}
+.box-row {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 16px;
```

Comment:
Consider using `padding: 16px` shorthand here and throughout your styles rather than specifying for each side separately. If you want to have different values, you can still use the shorthand and just provide 4 different numbers, which will be applied clockwise starting from the top, e.g. `padding: 10px 20px 16px 24px`. Same goes for margins. Saves a lot of lines of css and a bit easier to read, imo.